client/src/components/pages/Login/index.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component, Fragment } from "react";
+import { BrowserRouter as Router, Link } from "react-router-dom";
+import { Form, Icon, Input, Checkbox } from "antd";
+
+import Button from "components/utils/Button";
+import "./style.css";
+
+class LoginPage extends Component {
```

Comment:
just a note. 
here you don't need to class component you can just use function component. 
and btw function component faster than class component